ibc_test.go

```diff
@@ -100,7 +101,7 @@ func TestIBCHandshake(t *testing.T) {
 	msg := IBCInstantiateMsg{
 		ReflectCodeID: REFLECT_ID,
 	}
-	ires, _, err := vm.Instantiate(checksum, env, info, toBytes(t, msg), store, *goapi, querier, gasMeter1, TESTING_GAS_LIMIT)
+	ires, _, err := vm.Instantiate(checksum, env, info, toBytes(t, msg), store, *goapi, querier, gasMeter1, TESTING_GAS_LIMIT, &deserCost)
```

Comment:
Why do you use a pointer here? Can't we just copy that by value?

Reply:
I had weird problems with passing structs around earlier, but that must have been something weird I did. It works!